### PR TITLE
Add registry contract

### PR DESCRIPTION
**Pull request by grorge123**
Implement features:
- Query node
- Create node
- Update node

### CODE REVIEW

File: contracts/Registry.sol [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * @dev OtomicMarket logic
+ */
+contract Registry is Ownable {
+    
+    
+    struct Node {
+        string name;
+        string description;
+        string url;
+        bool availability;
```
Comment (juntao) [inline contracts/Registry.sol]: Let's change `availability` to `status`

File: contracts/Registry.sol [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        address creator;
```
Comment (juntao) [inline contracts/Registry.sol]: I am not sure why we need a `creator` here? The node is always mapped to an `address` in this contract. I think the `address` key is the address that owns (or created) the node?

File: contracts/Registry.sol (hunk 1 above)
Comment (grorge123) [reply contracts/Registry.sol]: I have fixed it. Please check thx.

File: contracts/Registry.sol (hunk 2 above)
Comment (grorge123) [reply contracts/Registry.sol]: I have fixed it. Please check thx.

**Review by juntao — APPROVED** [review]

**Pull request merged**